Services/RequestServer/ConnectionFromClient.cpp

```diff
@@ -447,7 +446,6 @@ void ConnectionFromClient::start_request(i32 request_id, ByteString const& metho
             }
 
             auto formatted_address = resolve_opt_builder.to_byte_string();
-            g_dns_cache.set(host, formatted_address);
```

Comment:
Nononono, this thing is holding on to the strings, without this the next access will do a UAF read

Reply:
Oh... I'll remove that.